nitro-protocol/contracts/interfaces/IForceMoveApp.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.4;
+pragma experimental ABIEncoderV2;
+
+/**
+ * @dev The IForceMoveApp interface calls for its children to implement an application-specific validTransition function, defining the state machine of a ForceMove state channel DApp.
+ */
+interface IForceMoveApp {
+    struct VariablePart {
+        bytes outcome;
+        bytes appData;
+    }
```

Comment:
@nksazonov This struct needs updating, too, I think?

Take a look at 744dbbef0ef22ba8085d60b8b0a8b468f038bc16 and 3b4bfbae06a4e5942ca2d88d80c2af1cea15740c

Reply:
Yeah, I've also realized it needs updating to support `latestSupportedState`.